src/Components/ProductCard/ProductCard.scss

```diff
@@ -159,6 +159,50 @@
     }
 }
 
-@media (max-width: 374px) {
-    
+.productcard.popular {
+    .productcard__image {
+        height: 148px;
+        margin-top: 6px;
+    }
+
+    .productcard__info {
+        height: 135px;
+        padding: 20px 10px 0px 0px;
+    }
+
+    .productcard__colors {
+        margin-left: -4px;
+    }
+
+    .colors__dot {
+        display: flex;
+        height: 10px;
+        width: 10px;
+        background-color: black;
+        border-radius: 50%;
+        visibility: visible;
+    }
+
+    .colors__border {
+        display: flex;
+        height: 15px;
+        width: 15px;
+        border-radius: 50%;
+        border: 1px solid grey;
+        justify-content: center;
+        align-items: center;
+        margin-right: 2px;
+        visibility: hidden;
+    }
+
+    .colors__border--active {
+        display: flex;
+        height: 15px;
+        width: 15px;
+        border-radius: 50%;
```

Comment:
please avoid of re-declaring css rules. 
border-radius: 50%; is specified in parent scope (line 80 in this file) as well as many other properties

Reply:
Done.